docs/quickstart-docker.md

```diff
@@ -0,0 +1,143 @@
+# Start a Docker container and take a backup
+
+Percona XtraBackup works in combination with a MySQL-compatible database, Percona Server for MySQL, in our scenario. To use Percona XtraBackup, you need to run it in its own Docker container and connect it to the Percona Server for MySQL container. As soon as you [start Percona Server for MySQL in a docker container](https://docs.percona.com/percona-server/8.0/quickstart-docker.html) and [create a database and table in Percona Server for MySQL](https://docs.percona.com/percona-server/8.0/quickstart-database.html), you are ready to backup your databes with Percona XtraBackup.
+
+The following steps create a Docker volume, start Percona XtraBackup in a Docker continer, take and prepare a backup of Percona Server for MySQL database.
+{.power-number}
+
+1. Create a Docker volume
+
+    One of the benefits of using a Docker volume for persistent storage with a database is that it allows you to keep your data safe and accessible across different containers. For example, if you want to upgrade your database image to a newer version, you can stop the old container and start a new one with the same volume attached without losing any data. Another benefit is that you can quickly backup and restore your data using the docker cp command or mount the volume to another container.
+
+    ```{.bash data-prompt="$"}
+    $ docker volume create backupvol
+    ```
+    
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+        backupvol
+        ```
+
+2. Start Percona XtraBackup 8.0.34 in a container, take and prepare a backup
+
+    To use the `docker run` command, specify the name or ID of the image you want to use and, optionally, some flags and arguments that modify the container's behavior. 
+
+    The command has the following options:
+
+    | Option | Description |
+    |---|---|
+    | `--name`  |  Provides a name to the container. If you do not use this option, Docker adds a random name.  |
+    | `--volumes-from` | Refers to Percona Server for MySQL and indicates that you intend to use the same data as the `psmysql` container.|
+    | `it` | Interacts with the container and be a pseudo-terminal. |
+    | `--user root` | Sets the user to root inside the container. This option is required to access the MySQL data directory and run the xtrabackup command. |
+    |`backup_8034` |Indicates the directory inside the container where the backup files are stored. |
+    | `-v` | Mounts a volume from the host machine or another container to the container that is being run. |
```

Comment:
more clarity. what does another container to the container mean?

Reply:
updated